exercises/roman-numerals/canonical-data.json

```diff
@@ -1,74 +1,92 @@
 {
    "cases": [
       {
+         "description": "1 is a single I",
          "number" : 1, 
          "expected": "I"
       },
       {
+         "description": "2 is two I's",
          "number" : 2, 
          "expected": "II"
       },
       {
+         "description": "3 is three I's",
          "number" : 3, 
          "expected": "III"
       },
       {
+         "description": "4, being 5 - 1, is IV",
```

Comment:
Hmmm, I a bit torn on this one. Perhaps we should first do all of the individual letter case (I, V, X, etc.) and then move on to the edge cases? I think that would make more sense for this particular problem domain, even though it can seem a bit odd at first.